index.js

```diff
@@ -16,22 +16,46 @@ module.exports = (api, configOptions) => {
     let options = configOptions.pluginOptions.s3Deploy
 
     // Check for environment overrides of the options in vue.config.js.
+    options.region = process.env.VUE_APP_S3D_REGION || options.region
     options.bucket = process.env.VUE_APP_S3D_BUCKET || options.bucket
+    options.createBucket = process.env.VUE_APP_S3D_CREATE_BUCKET || options.createBucket
+    options.uploadConcurrency = process.env.VUE_APP_S3D_UPLOAD_CONCURRENCY || options.uploadConcurrency
+
+    options.staticHosting = process.env.VUE_APP_S3D_STATIC_HOSTING || options.staticHosting
+    options.staticIndexPage = process.env.VUE_APP_S3D_STATIC_INDEX_PAGE || options.staticIndexPage
+    options.staticErrorPage = process.env.VUE_APP_S3D_STATIC_ERROR_PAGE || options.staticErrorPage
+    options.staticWebsiteConfiguration = process.env.VUE_APP_S3D_STATIC_WEBSITE_CONFIGURATION || options.staticWebsiteConfiguration
+
     options.assetPath = process.env.VUE_APP_S3D_ASSET_PATH || options.assetPath
+    options.assetMatch = (process.env.VUE_APP_S3D_ASSET_MATCH || options.assetMatch)
     options.deployPath = process.env.VUE_APP_S3D_DEPLOY_PATH || options.deployPath
-    options.region = process.env.VUE_APP_S3D_REGION || options.region
+    options.acl = process.env.VUE_APP_S3D_ACL || options.acl
+
     options.pwa = process.env.VUE_APP_S3D_PWA || options.pwa
-    options.uploadConcurrency = process.env.VUE_APP_S3D_UPLOAD_CONCURRENCY || options.uploadConcurrency
+    options.pwaFiles = process.env.VUE_APP_S3D_PWA_FILES || options.pwaFiles
+
     options.enableCloudfront = process.env.VUE_APP_S3D_ENABLE_CLOUDFRONT || options.enableCloudfront
     options.cloudfrontId = process.env.VUE_APP_S3D_CLOUDFRONT_ID || options.cloudfrontId
     options.cloudfrontMatchers = process.env.VUE_APP_S3D_CLOUDFRONT_MATCHERS || options.cloudfrontMatchers
 
+    // parse and correct for boolbean vars passed as strings
+    Object.keys(options).forEach(key => {
+      let option = options[key]
```

Comment:
Don't need line 43-44. Just wrap it in `if (options[key])`

Reply:
Whoops, yup will do